.clang-format

```diff
@@ -2,22 +2,22 @@
 Language:        Cpp
 # BasedOnStyle:  Google
 AccessModifierOffset: -1
-AlignAfterOpenBracket: Align
-AlignConsecutiveAssignments: true
-AlignConsecutiveDeclarations: true
+AlignAfterOpenBracket: DontAlign # TODO
+AlignConsecutiveAssignments: false
```

Comment:
I think DontAlign is correct here

Reply:
Again I'm in favor of adopting a standard and sticking to it. 

Don't align means that if we break function arguments they should appear on the next line indented.

```c
int SomeLongFunctionName(firstArg,
    SecondArg, ThirdArg);
```

This has the advantage of not changing on a function rename.

The other options are `Align`, `AlwaysBreak` and `BlockIndent` which you can see [here](https://clang.llvm.org/docs/ClangFormatStyleOptions.html) under AlignAfterOpenBracket. 

I don't have a strong preference. But I don't see a clearly correct alternative choice. If you do let me know and I'll change it.

Reply:
I'm saying I think DontAlign is correct. I'm commenting on this because you put a "TODO" next to it, suggesting you weren't sure about the setting / might want more opinions. Unless your TODOs mean something else?